### PR TITLE
added admin ui 

### PR DESCRIPTION
**Pull request by Homa**

### CODE REVIEW

File: client/app/admin/scripts/workstreamController.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
+'use strict';
+
+angular.module('elixirApp')
+.controller('WorkstreamIndexController', ['$scope', 'Workstream', 'Task', function($scope, Workstream, Task) {
+
+    // add workstream form init
+    $scope.showNewWorkstreamLink = true;
+    $scope.workstream = {};
```
Comment (kylebuch8) [inline client/app/admin/scripts/workstreamController.js]: Why don't you initialize the object like:
```
$scope.workstream = {
    color: 'red'
};

File: client/app/admin/views/index.html [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+          </ul>
+      </div>
+  </div>
+</nav>
+
+<div class="container">
+  <div class="row">
+    <!-- Add workstream -->
+    <div>
+      <a class="pull-right" ng-click="showNewWorkstreamForm=true;showNewWorkstreamLink=false" ng-show="showNewWorkstreamLink" style="height:34px;">Add New Workstream</a>
```
Comment (kylebuch8) [inline client/app/admin/views/index.html]: Can we get the inline style moved to the stylesheet?

File: client/app/admin/views/index.html [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+      <div ng-show="items.$resolved && items.length == 0">
+        <p>Use the 'Add New Workstream' button to create a new workstream!</p>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <table class="table table-cp table-striped" summary="A table showing people's names, their ages, and their favorite colors">
+      <thead>
+        <tr>
+          <th id="th-name" scope="col" style="width:200px;">Workstream</th>
```
Comment (kylebuch8) [inline client/app/admin/views/index.html]: Can we get the inline styles added to the stylesheet?

**Comment by Homa** [comment]
after review decided to close this pull request and send another one...

**Pull request closed without merging**